modules/admin/admin-impl/src/main/java/com/enonic/xp/admin/impl/rest/resource/content/CmsResourceDynamicFeature.java

```diff
@@ -0,0 +1,35 @@
+package com.enonic.xp.admin.impl.rest.resource.content;
+
+import java.util.List;
+
+import javax.ws.rs.container.DynamicFeature;
+import javax.ws.rs.container.ResourceInfo;
+import javax.ws.rs.core.FeatureContext;
+import javax.ws.rs.ext.Provider;
+
+import com.enonic.xp.admin.impl.rest.resource.content.page.PageResource;
+import com.enonic.xp.admin.impl.rest.resource.content.page.PageTemplateResource;
+import com.enonic.xp.admin.impl.rest.resource.content.page.fragment.FragmentResource;
+import com.enonic.xp.admin.impl.rest.resource.schema.content.ContentTypeResource;
+import com.enonic.xp.admin.impl.rest.resource.schema.xdata.XDataResource;
+import com.enonic.xp.jaxrs.JaxRsComponent;
+
+
+@Provider
+public final class CmsResourceDynamicFeature
+    implements DynamicFeature, JaxRsComponent
+{
+    private final List<Class<? extends JaxRsComponent>> SUPPORTED_RESOURCES =
+        List.of( ContentResource.class, XDataResource.class, PageTemplateResource.class, ContentImageResource.class,
```

Comment:
Looks strange...
Why capital case, but not static?
But I agree it should not be static :)
It is also Set, not list, because there are no duplicates